integration_tests/helpers/walletFFIClient.js

```diff
@@ -74,6 +74,10 @@ class WalletFFIClient {
     return this.wallet.getBalance();
   }
 
+  pollBalance() {
+    return this.wallet.pollBalance();
```

Comment:
I think these methods would be more aptly named `getBalanceFromWallet()` to minimize confusion (see related comment)